### ISSUE TITLE
apm test Fails on Windows

### ISSUE THREAD

**Issue by Arcanemagus** [atom/atom#9277]
This has to have been already reported, but I can't seem to find it...

When running on Windows it seems that `apm test` will report that the tests failed regardless of the actual state of the tests.

For example if the spec consists entirely of this:
```CoffeeScript
describe "Test package", ->
  it "has one valid test", ->
    expect(true).toBe true
```

Running `apm test` on that produces the following output:
```
PS C:\Users\Landon\github\test> apm test
[48536:1024/140634:WARNING:accelerator_util.cc(185)] Invalid accelerator token: command
[48536:1024/140634:WARNING:accelerator_util.cc(185)] Invalid accelerator token: command
[48536:1024/140634:WARNING:accelerator_util.cc(185)] Invalid accelerator token: command
[48536:1024/140634:WARNING:accelerator_util.cc(185)] Invalid accelerator token: command
[49320:1024/140634:INFO:renderer_main.cc(200)] Renderer process started
[48536:1024/140636:INFO:CONSOLE(46)] "Window load time: 1020ms", source: file:///C:/Users/Landon/AppData/Local/atom/app-1.0.19/resources/app.asar/static/index.js (46)
.

Finished in 0.079 seconds
1 test, 1 assertion, 0 failures, 0 skipped


Tests failed
```

I have seen this behavior on Windows 8.1 x64, Windows 10 x64, and Windows Server 2012 R2 (AppVeyor), all on Atom 1.0.19. The exact same tests work perfectly fine on Linux and OS X builds.

If there is any further information you need please let me know.

**Comment by thomasjo** [comment]
@atom/feedback This has got to be a duplicate? I was sure @50Wliu opened an issue for this a little while ago? I can't seem to find it though (if it exists).

**Comment by 50Wliu** [comment]
I don't actually remember opening an issue for it :/.

**Comment by joefitzgerald** [comment]
@Arcanemagus Thanks for reporting this, this is definitely an issue.

**Comment by Arcanemagus** [comment]
Not that I was expecting it to since it wasn't mentioned here, but I thought that I would report that v1.2.0 does not fix this issue.
(It does thankfully get rid of all the error spam if Atom is already running though :smile:)

**Comment by joefitzgerald** [comment]
@arcanemagus this is fixed via the commit to upgrade to Electron 4.3.3. Be on the lookout for that to hit beta and stable in the next releases.

**Comment by joefitzgerald** [comment]
https://github.com/atom/atom/commit/0dd84703c12007de81cf3085fe514db8c2c1f4d7

**Comment by maxbrunsfeld** [comment]
Thanks @joefitzgerald. I'm going to close this out as it should be fixed in `v1.3.0` (or `v1.3.0-beta0`, if you're on the beta channel)

**Comment by Arcanemagus** [comment]
Awesome, thanks!
